daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf2.proto

```diff
@@ -355,18 +350,26 @@ enum BuiltinFunction {
 
   /* Dev Builtins */
 
-  SCALE_BIGNUMERIC = 1001;             // *Available in versions >= 2.dev
-  PRECISION_BIGNUMERIC = 1002;         // *Available in versions >= 2.dev
-  ADD_BIGNUMERIC = 1003;               // *Available in versions >= 2.dev
-  SUB_BIGNUMERIC = 1004;               // *Available in versions >= 2.dev
-  MUL_BIGNUMERIC = 1005;               // *Available in versions >= 2.dev
-  DIV_BIGNUMERIC = 1006;               // *Available in versions >= 2.dev
-  SHIFT_RIGHT_BIGNUMERIC = 1007;       // *Available in versions >= 2.dev
-  BIGNUMERIC_TO_NUMERIC = 1008;        // *Available in versions >= 2.dev*
-  NUMERIC_TO_BIGNUMERIC = 1009;        // *Available in versions >= 2.dev
-  BIGNUMERIC_TO_TEXT = 1010;           // *Available in versions >= 2.dev
-
-  TYPE_REP_TYCON_NAME = 1011;          // *Available in versions >= 2.dev*
+
+  TEXTMAP_EMPTY = 1001;
+  TEXTMAP_INSERT = 1002;
+  TEXTMAP_LOOKUP = 1003;
+  TEXTMAP_DELETE = 1004;
+  TEXTMAP_TO_LIST = 1005;
+  TEXTMAP_SIZE = 1006;
```

Comment:
Should it also say available in >= 2.dev?

Reply:
Yes.